tsconfig.base.json

```diff
@@ -4,6 +4,7 @@
 		"lib": ["ESNext", "DOM"],
 		"emitDecoratorMetadata": false,
 		"target": "ESNext",
-		"module": "ESNext"
+		"module": "ESNext",
```

Comment:
```suggestion
```

`module` `node16` (the default from extended) should be fine now that we use `moduleResolution` `node16` as well.

(before with resolution set to `node` and `module` had to be set to `ESNext` for TS to output `import` syntax over `require` syntax)